python/veles/common/base.py

```diff
@@ -36,6 +36,44 @@ def bytes(self):
     def to_bytes(self):
         return self.bytes
 
+    @staticmethod
+    def generate_cpp_code():
+        code = '''
```

Comment:
To jest pojedynczy kawałek kodu bez żadnych parametrów z pythona, proponuję go napisać po prostu w osobnym pliku C++ i się nie bawić.  Tak samo z wszelkimi innymi ext typami (nie będzie ich dużo).